entity-service/src/main/kotlin/com/egm/stellio/entity/config/WebSecurityConfig.kt

```diff
@@ -34,4 +39,9 @@ class WebSecurityConfig {
 
         return http.build()
     }
+
+    @Bean
+    fun jwtDecoder(): ReactiveJwtDecoder {
```

Comment:
Explain why you had to do this. And I'm not in favor of modifying something in the main source tree for testing purposes only.

Reply:
Since WebFluxTest cannot detect the custom security configuration, we need to Import it explicitly.
The tests failed with (consider defining a bean of type ReactiveJwtDecoder in your configuration).
That's why i configured the bean.
We need to import the security configuration (in order to disable csrf) otherwise tests will fail with the status 403 forbidden.
We have another option: Define WebSecurityTestConfig in wich we disable csrf but we cannot test the anonymous access.